CONTRIBUTING.md

```diff
@@ -0,0 +1 @@
+Contirbuting.md is available on the [wiki](https://github.com/musescore/MuseScore/wiki/Contributing.md)
```

Comment:
Typo, Contirbuting.md vs Contributing.md

Reply:
Also @shoogle's comment hasn't been acted upon, yet the conversation had ben maked resolved?

Reply:
Sorry about that. It's been changed and pushed, and commits have been squashed.

Reply:
Great, thanks!